### ISSUE TITLE
Pressing "Enter" in the project name field at the preview stage should create the project

### ISSUE THREAD

**Issue by wetneb** [OpenRefine/OpenRefine#3137]
**Is your feature request related to a problem or area of OpenRefine? Please describe.**

When previewing an importer's results, there is a text field to enter the project name. When pressing Enter in this field, I would expect the project to be created (without having to click the "Create Project" button).

![image](https://user-images.githubusercontent.com/309908/91545213-1a244a80-e921-11ea-938a-b6e40b9404ad.png)

**Comment by thadguidry** [comment]
Agree.  The current Enter key binding to the Update Preview button is not needed... since all the options on importer are bound to the Update Preview already, I think?  Instead, hitting Enter key for the Create Project flow makes more sense.

**Comment by rachitmanas** [comment]
Hey. I would like to work on this issue. Can someone please give me an abstract idea of where I can find the code relating to this issue?

**Comment by wetneb** [comment]
I can tell you my method to find it:
* Identify a bit of text in this part of the UI that should identify it across the entire codebase. There are not that many places where we can create projects, so the "Create Project" text on the button looks like a good candidate.
* Search for this string in the repository. I do this with `grep` but you can do it with all sorts of other tools (your IDE, for instance): `grep -R "Create Project" .`
* In the results, I see: `./main/webapp/modules/core/langs/translation-en.json:    "core-buttons/create-project": "Create Project &raquo;",` - that is the string you see in the screenshot! Unfortunately it is not input directly in the relevant JS/HTML code, but in a translation file instead. That is not a problem: we can keep searching using the translation identifier `core-buttons/create-project`.
* Therefore I run `grep -R "core-buttons/create-project" .`, and I find:  `./main/webapp/modules/core/scripts/index/default-importing-controller/parsing-panel.js:  this._parsingPanelElmts.nextButton.html($.i18n('core-buttons/create-project'));`. That is the Javascript file we are looking for!
* From there, you can just open the javascript file and you will also find the mention of the HTML template that is used for this part of the UI.

With this technique you should be able to find your way in quite a lot of code bases, not just OpenRefine's.

**Comment by rachitmanas** [comment]
Oh this is so helpful and smart trick. Thank You for explaining in such details.
I will be solving this issue now.

**Comment by leoshrey** [comment]
hi @wetneb , I am working on this issue, can you please assign me?

**Comment by leoshrey** [comment]
Hi @wetneb , can you please review the PR linked ?

**Comment by singhakshita** [comment]
Hey is this issue still open  (since its mentioned  that will close this issue on 13 Dec 2020) I m not very sure 
can you please assign me this .

**Comment by wetneb** [comment]
No, @leoshrey seems to be still working on this issue.

**Comment by johnvl2** [comment]
@wetneb According to the latest comments in pull request 3406, it seems like there is still work to be done for this? If so, I can look into it. Thanks.

**Comment by wetneb** [comment]
@johnvl2 thanks! The floor is yours.

**Comment by khadijm** [comment]
hi @wetneb please can i work on this issues, thanks

**Comment by khadijm** [comment]
i am getting this white page

![Screenshot from 2022-04-02 10-14-03](https://user-images.githubusercontent.com/88087734/161376418-e84dbb90-1074-48c1-acf8-73ea0d6083f1.png)


``` 

 ./refine
Using refine.ini for configuration
-------------------------------------------------------------------------------------------------
You have 15908M of free memory.
Your current configuration is set to use 1400M of memory.
OpenRefine can run better when given more memory. Read our FAQ on how to allocate more memory here:
https://docs.openrefine.org/manual/installing#increasing-memory-allocation
-------------------------------------------------------------------------------------------------

10:13:03.302 [            refine_server] Starting Server bound to '127.0.0.1:3333' (0ms)
10:13:03.310 [            refine_server] refine.memory size: 1400M JVM Max heap: 1468006400 (8ms)
10:13:03.340 [            refine_server] Initializing context: '/' from '/home/bebeto/intellij/openrefine/OpenRefine/main/webapp' (30ms)
10:13:03.380 [            refine_server] Starting autoreloading scanner...  (40ms)
Gtk-Message: 10:13:05.566: Failed to load module "canberra-gtk-module"
[20448:20448:0402/101305.797879:ERROR:sandbox_linux.cc(377)] InitializeSandbox() called with multiple threads in process gpu-process.
Opening in existing browser session.
10:13:06.179 [                   refine] Starting OpenRefine 3.6-SNAPSHOT [7ab6eed]... (2799ms)
10:13:06.179 [                   refine] initializing FileProjectManager with dir (0ms)
10:13:06.179 [                   refine] /home/bebeto/.local/share/openrefine (0ms)


```

**Comment by antoine2711** [comment]
@khadijm: how is it going with this issue?

Regards, Antoine
